### PR TITLE
Documentation of method After

### PR DESCRIPTION
**Pull request by israeleriston**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by israeleriston** [comment]
Issue https://github.com/fastify/fastify/issues/841

File: docs/Server-Methods.md [hunk 1]
```diff
@@ -25,6 +25,17 @@ fastify.ready().then(() => {
 })
 ```
 
+<a name="after"></a>
+### after
+An function called when all routes have been loaded
```
Comment (jsumners) [inline docs/Server-Methods.md]: "Invoked when all routes..."

**Comment by israeleriston** [comment]
I will change

File: docs/Server-Methods.md (hunk 1 above)
Comment (allevo) [reply docs/Server-Methods.md]: "Invoked when all plugins and routes ..."

File: docs/Server-Methods.md [hunk 2]
```diff
@@ -25,6 +25,17 @@ fastify.ready().then(() => {
...
+and executed before the method ``` fastify.ready ```
+
+```js
+fastify.after(err => {
+  console.log('successfully after loaded all routes!')
```
Comment (allevo) [inline docs/Server-Methods.md]: `'successfully after loaded all routes!'` => `'this plugin is loaded!'`

**Comment by israeleriston** [comment]
Great, thanks for the feedback @allevo @jsumners

File: docs/Server-Methods.md [hunk 3]
```diff
@@ -25,6 +25,17 @@ fastify.ready().then(() => {
 })
 ```
 
+<a name="after"></a>
+### after
+Invoked when all plugins and routes have been loaded
+and executed before the method ``` fastify.ready ```
```
Comment (mcollina) [inline docs/Server-Methods.md]: This is partially correct. The callback passe to `after(cb)` will be invoked when the current plugin and all the plugin that has been registered under him have finished loading. At the top level, this is right before ready.

(also, remove the space and use single backticks if the code is inline)

Comment (israeleriston) [reply docs/Server-Methods.md]: @mcollina  is more explicit I say "The callback pass to after (cb)" or just "Invoked when the current plugin and all ..." which is more clear?

File: docs/Server-Methods.md [hunk 4]
```diff
@@ -6,6 +6,17 @@
 #### server
 `fastify.server`: The Node core [server](https://nodejs.org/api/http.html#http_class_http_server) object as returned by the [**`Fastify factory function`**](https://github.com/fastify/fastify/blob/master/docs/Factory.md).
 
+<a name="after"></a>
+### after
+Invoked when the current plugin and all the plugin that has been registered under
```
Comment (delvedor) [inline docs/Server-Methods.md]: I would rephrase it as:
Invoked when the current plugin and all the plugin that has been registered inside it have finished loading. It is always executed before the method `fastify.ready`.

File: docs/Server-Methods.md [hunk 5]
```diff
@@ -6,6 +6,17 @@
...
+him have finished loading and is executed before the method ``` fastify.ready ```
+
+```js
+fastify.after(err => {
+  console.log('this plugin is loaded!')
+})
+```
```
Comment (delvedor) [inline docs/Server-Methods.md]: I think you should add also `register` and `ready`, in this way the user can try it and it will be more clear.
```js
fastify
  .register((instance. opts, next) => {
    console.log('Current plugin')
    next()
  })
  .after(err => {
    console.log('After current plugin')
  })
  .register((instance. opts, next) => {
    console.log('Next plugin')
    next()
  })
  .ready(err => {
    console.log('Everything has been loaded')
  })
```

File: docs/Server-Methods.md [hunk 6]
```diff
@@ -6,6 +6,30 @@
 #### server
 `fastify.server`: The Node core [server](https://nodejs.org/api/http.html#http_class_http_server) object as returned by the [**`Fastify factory function`**](https://github.com/fastify/fastify/blob/master/docs/Factory.md).
 
+<a name="after"></a>
+### after
```
Comment (delvedor) [inline docs/Server-Methods.md]: Can you add another `#` here?

Comment (israeleriston) [reply docs/Server-Methods.md]: yeah! done @delvedor

File: docs/Server-Methods.md [hunk 7]
```diff
@@ -6,6 +6,30 @@
 #### server
 `fastify.server`: The Node core [server](https://nodejs.org/api/http.html#http_class_http_server) object as returned by the [**`Fastify factory function`**](https://github.com/fastify/fastify/blob/master/docs/Factory.md).
 
+<a name="after"></a>
+#### after
+Invoked when the current plugin and all the plugin
```
Comment (jsumners) [inline docs/Server-Methods.md]: "and all the plugin" => "and all the plugins"

File: docs/Server-Methods.md [hunk 8]
```diff
@@ -6,6 +6,30 @@
 #### server
 `fastify.server`: The Node core [server](https://nodejs.org/api/http.html#http_class_http_server) object as returned by the [**`Fastify factory function`**](https://github.com/fastify/fastify/blob/master/docs/Factory.md).
 
+<a name="after"></a>
+#### after
+Invoked when the current plugin and all the plugin 
+that has been registered inside it have finished loading.
```
Comment (jsumners) [inline docs/Server-Methods.md]: "that has been registered inside" => "that have been registered within"

File: docs/Server-Methods.md [hunk 9]
```diff
@@ -6,6 +6,30 @@
...
+It is always executed before the method ``` fastify.ready ```.
```
Comment (jsumners) [inline docs/Server-Methods.md]: There should only be one pair of backticks surrounding "fastify.ready".

Comment (israeleriston) [reply docs/Server-Methods.md]: done @jsumners
